include/osqp_api_constants.h

```diff
@@ -125,4 +126,4 @@ extern const char * OSQP_ERROR_MESSAGE[];
 }
 # endif // ifdef __cplusplus
 
-#endif // ifndef CONSTANTS_H
+#endif // ifndef OSQP_API_CONSTANTS_H
```

Comment:
How about prepending all the constants by `OSQP_`? This is also related to https://github.com/osqp/osqp/pull/260.